src/js/abstract/base-component.js

```diff
@@ -182,7 +246,7 @@ export default class BaseComponent extends HTMLElement {
           });
         }
 
-        const items = template(getAttributes(this), this.childrenFragment);
+        const items = template(this._props, this.childrenFragment);
```

Comment:
i have a mixed feeling trying to reactify standard web components. I like this solution, dont get me wrong, but im just a bit scared :) U can ignore this comment :)